aws-observability/templates/sumologic_observability.master.template.yaml

```diff
@@ -127,10 +127,10 @@ Metadata:
         default: "Select the Sumo Logic CloudWatch Logs Sources"
       Section7bLambdaCloudWatchLogsSourceUrl:
         default: "Existing Sumo Logic Lambda CloudWatch Logs Source API URL"
-      Section7cAutoSubscribeLogGroupsLambdaOptions:
+      Section7cAutoSubscribeLogGroupsOptions:
         default: "Subscribe log groups to Sumo Logic CloudWatch Logs Forwarder"
```

Comment:
Subscribe log groups to Destination (Lambda or kinesis firehose delivery stream)